tasks/unicorn.yml

```diff
@@ -2,10 +2,10 @@
 - name: Unicorn - Install unicorn as gem.
   gem:
     name: unicorn
-    version: 5.4.1
+    version: 6.1.0
     executable: "{{ redmine_home }}/.rbenv/shims/gem"
     user_install: no
-    install_dir: "{{ redmine_home }}/.rbenv/versions/{{ unicorn_ruby_version }}/lib/ruby/gems/2.6.0/"
+    install_dir: "{{ redmine_home }}/.rbenv/versions/{{ unicorn_ruby_version }}/lib/ruby/gems/{{ unicorn_ruby_version }}/"
```

Comment:
unicorn_ruby_version should be initialized in default/main.yml to avoid breaking change

Reply:
ok